lib/identity_cache/configuration_dsl.rb

```diff
@@ -171,31 +171,35 @@ def cache_has_one(association, options = {})
       # * by: Other attribute or attributes in the model to keep values indexed. Default is :id
       # * unique: if the index would only have unique values. Default is true
       def cache_attribute(attribute, options = {})
+        cache_attribute_by_alias(attribute.inspect, attribute, options)
```

Comment:
Because I needed something that would be lazily evaluated. It is used in the `instance_eval` string with `attribute_dynamic_fetcher(#{attribute}`.